### DISCUSSION TITLE
MultiplayerCore v1.3.1

### DISCUSSION THREAD

**Discussion by michael-r-elp** [EnderdracheLP/MultiplayerCore.Quest#13] [General]
## What's Changed
- Changed the way the mod initializes, it should now initialize much later.

**Full Changelog**: https://github.com/EnderdracheLP/MultiplayerCore.Quest/compare/1.3.0...1.3.1

As you all know, due to issues where some peoples game crash on startup with MultiplayerCore installed, 
we, for the time being, decided to have MultiplayerCore be taken out of the Core Mods list

This patch of MultiplayerCore was made to potentially address that issue by changing the way the mod initializes.
The mod now initializes only when pressing the "Modded Online" Button.

This should hopefully help people affected by the crash to play normally and only experience the crash when attempting to enter Multiplayer.
Since we're having a hard time finding people internally that experienced the mentioned crashes, 
we decided to publish this build here, in hopes of gathering feedback from a wide range of quest players

Please try this build out and take some time to fill out our form to tell us about your experience

TLDR:
Some people experienced startup crashes with MultiplayerCore
This build helps possibly mitigate the issue
We need your feedback

__*Feedback Form:*__ https://forms.gle/DwesiwAFUoAe9MDn6

<hr /><em>This discussion was created from the release <a href='https://github.com/EnderdracheLP/MultiplayerCore.Quest/releases/tag/1.3.1'>MultiplayerCore v1.3.1</a>.</em>
